rcl/include/rcl/context.h

```diff
@@ -240,6 +240,29 @@ RCL_WARN_UNUSED
 rcl_context_instance_id_t
 rcl_context_get_instance_id(rcl_context_t * context);
 
+/// Returns the context domain id.
+/**
+ * \pre If context is uninitialized, then it is undefined behavior.
+ *
+ * <hr>
+ * Attribute          | Adherence
+ * ------------------ | -------------
+ * Allocates Memory   | No
+ * Thread-Safe        | No
+ * Uses Atomics       | No
+ * Lock-Free          | No
+ *
+ * \param[in] context object from which the domain id should be retrieved.
+ * \param[out] domain_id output variable where the domain id will be returned.
+ * \return RCL_RET_INVALID_ARGUMENT if `context` is invalid, or
+ *  RCL_RET_INVALID_ARGUMENT if `domain_id` is `NULL`, or
+ *  RCL_RET_OK if the domain id was correctly retrieved.
```

Comment:
```suggestion
 * \return RCL_RET_INVALID_ARGUMENT if `domain_id` is `NULL`, or
 * \return RCL_RET_OK if the domain id was correctly retrieved.
```

Reply:
Ups, I forgot how to write doxygen correctly :joy:.
See https://github.com/ros2/rcl/pull/718/commits/7aa5594694b54a3aefe76a9b2d6d3b74af94e075.